cves/kernel/CVE-2013-3224.yml

```diff
@@ -371,8 +398,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: | 
+    The memory buffers/other closely related items such as the message variable 
+    should have been cleared before the socket was shutdown, but in this case, it wasn't. 
```

Comment:
This doesn't make it seem like they moved code around/had to change the order of things, unless they were trying to clear the memory after the socket shutdown.

Reply:
Thanks. Looks like I did not make that very clear in the note; I have updated it. 
In this case, the developers forgot to set the length of the message to 0 when the socket was already shutting down (in which , the function would simply return 0, but the memory was never reset as the message length was set to 0 only after the socket was shutting down, leaking some information to the stack memory.) They had to make sure to set the message length to 0 , before the socket was shutdown, not after.